ci-conda.Dockerfile

```diff
@@ -3,6 +3,7 @@ ARG LINUX_VER=notset
 ARG PYTHON_VER=notset
 ARG YQ_VER=notset
 ARG AWS_CLI_VER=notset
+ARG MINIFORGE_VER=notset
```

Comment:
Oh sorry, should have caught this.... this `ARG` needs to come down AFTER that first `FROM`.

Reply:
So I'd thought this was just the issue where an `ARG` needs to be restated before statement referencing it, but looks like it actually runs deeper... `docker` / `docker buildx` don't support interpolation with build args in `COPY --from` statements:

* https://github.com/moby/moby/issues/34482
* https://github.com/docker/buildx/issues/2352
* https://stackoverflow.com/a/51983295/3986677

So I just pushed https://github.com/rapidsai/ci-imgs/pull/247/commits/e296ecb5811ba889eca6d4de5a943ca4b3c94ddc to try a workaround... declaring the `miniforge` image in a `FROM` (to alias it to a hard-coded name), then referencing that hard-coded name again in the `COPY --from`. Let's see how that goes.

Reply:
How I tested this locally:

```shell
docker buildx build \
    --target miniforge-cuda \
    --build-arg AWS_CLI_VER=2.22.2 \
    --build-arg CODECOV_VER=9.1.1 \
    --build-arg CONDA_ARCH=linux64 \
    --build-arg CPU_ARCH=amd64 \
    --build-arg CUDA_VER=11.4.3 \
    --build-arg GH_CLI_VER=2.64.0 \
    --build-arg LINUX_VER=ubuntu20.04 \
    --build-arg MANYLINUX_VER=manylinux_2_31 \
    --build-arg MINIFORGE_VER=24.11.3-0 \
    --build-arg PYTHON_VER_UPPER_BOUND=3.11.0a0 \
    --build-arg REAL_ARCH=x86_64 \
    --build-arg SCCACHE_VER=0.7.7 \
    --build-arg YQ_VER=4.44.6 \
    -f ./ci-conda.Dockerfile \
    ./context
```